erpnext/agriculture/doctype/plant_batch/plant_batch.js

```diff
@@ -46,8 +49,38 @@ frappe.ui.form.on('Plant Batch', {
 				});
 			}, __("Create"));
 		}
-
-	}
+	},
+	split_plant_batch: function(frm) {
+		frappe.prompt([{
+			fieldname: 'split_count',
+			label: __('New Plant Batch Untracked Count'),
+			fieldtype: 'Int',
+			reqd: 1
+		},
+		{
+			fieldname: 'new_plant_batch_id',
+			label: __('New Plant Batch ID'),
+			fieldtype: 'Data',
+			reqd: 1
+		}],
+		(data) => {
+			if (frm.doc.untracked_count < data.split_count) {
+				frappe.throw(__("The Split count ({0}) should be less than or equal to the untracked quantity ({1})", [data.split_count.bold(), frm.doc.untracked_count.bold()]));
+			}
```

Comment:
If you're doing this on the server side, I'd say remove this validation from here